tests/runtests.py

```diff
@@ -47,6 +47,12 @@
 warnings.simplefilter("error", RuntimeWarning)
 # Ignore known warnings in test dependencies.
 warnings.filterwarnings("ignore", "'U' mode is deprecated", DeprecationWarning, module='docutils.io')
+# RemovedInDjango41Warning: Ignore MemcachedCache deprecation warning.
+warnings.filterwarnings(
+    'ignore',
+    'MemcachedCache is deprecated',
+    category=RemovedInDjango41Warning,
+)
```

Comment:
I haven't found a better way to keep tests for deprecated backend, it seems that a ` warnings.filterwarnings()` round-trip is necessary (here and in `MemcachedCacheDeprecationTests.test_warning()`).

Reply:
That seems unfortunate. Not sure I have any good suggestions though.

Reply:
Only option would be to not test it I think… 🤔

Reply:
Yeah, but that's not really an option as we need to support it in the deprecation period. I think it's fine.